### PR TITLE
[V2] fix: Rename parameters passed during v2 initialization

### PR DESCRIPTION
**Pull request by nearora-msft**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
The current name of the lease for azuredisk controller pod is "default". Changed it to a more suitable name.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by sunpa93** [comment]
@edreed Do we want to create the lease object under driver objectNamespace (which in AKS preview will be fixed to `kube-system) or `kube-system` by default?

File: pkg/azuredisk/azuredisk_v2.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ type DriverV2 struct {
 
 // NewDriver creates a driver object.
 func NewDriver(options *DriverOptions) CSIDriver {
-	return newDriverV2(options, *driverObjectNamespace, "default", "default", *heartbeatFrequencyInSec, *controllerLeaseDurationInSec, *controllerLeaseRenewDeadlineInSec, *controllerLeaseRetryPeriodInSec)
+	return newDriverV2(options, *driverObjectNamespace, "csi-azuredisk-node", "csi-azuredisk-controller", *heartbeatFrequencyInSec, *controllerLeaseDurationInSec, *controllerLeaseRenewDeadlineInSec, *controllerLeaseRetryPeriodInSec)
```
[the remark this answers is not in the document]
Comment (nearora-msft) [reply pkg/azuredisk/azuredisk_v2.go:98]: Ok, I'll take these as parameters and keep these names as default.

**Comment by edreed** [comment]
> @edreed @nearora-msft Do we want to create the lease object under driver objectNamespace (which in AKS preview will be fixed to `kube-system`) or `kube-system` by default?

Good point! The lease object itself should be in the same namespace as the pod itself and not the driver object namespace.

**Comment by edreed** [comment]
@nearora-msft The Helm package needs to be updated and the index rebuilt.

**Comment by edreed** [comment]
I believe the test failures will be addressed in commit [648e36f13e9a](https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/1370/commits/648e36f13e9a3d09ba4f1c9043a37f1d7eebf70c) when #1370 completes.

**Comment by edreed** [comment]
@nearora-msft The test failed due to the same `ConflictingUserInput` while trying to attach to two nodes in parallel. I am approving and will merge this change manually. I'll disable the test for now and investigate further.

/approve
/lgtm

**Comment by nearora-msft** [comment]
> @nearora-msft The test failed due to the same `ConflictingUserInput` while trying to attach to two nodes in parallel. I do not believe this related to your change. I am approving and will merge this change manually. I'll disable the test for now and investigate further.
> 
> /approve /lgtm

Sounds good, thanks!

**Pull request merged**